python/tchannel/handler.py

```diff
@@ -94,21 +99,29 @@ class TChannelResponse(object):
     """TChannel Response Wrapper"""
 
     __slots__ = ('_connection', '_request',
-                 'resp_msg', 'id')
+                 'resp_msg', 'id', 'body',
+                 'headers')
 
     def __init__(self, request, conn):
         self._connection = conn
         self._request = request
-        self.resp_msg = CallResponseMessage()
+        self.body = ""
+        self.headers = ""
         self.id = request.id
+        self.resp_msg = None
```

Comment:
Or are we not using this anymore? If we're not using it we should get rid of it. It's a little confusion that there's two ways to build a response.